storage/innobase/include/log0types.h

```diff
@@ -692,6 +692,10 @@ struct alignas(ut::INNODB_CACHE_LINE_SIZE) log_t {
   if lag between oldest_lsn and checkpoint_lsn exceeds max_checkpoint_lag. */
   lsn_t max_checkpoint_age_async;
 
+  /** When checkpoint age exceeds this value, user thread needs
+  to wait. The check is performed when a new query step is started. */
+  atomic_lsn_t max_checkpoint_age;
```

Comment:
Do u know where this variable is used? log_free_capacity(log) used may be? better to use this max_checkpoint_age variable then? 

And afaiu, this variable value, the max age doesn't change runtime right? (and it fixed value based on redo log file size?)